manifest/owo.Nytra/HeadlessToolPermissionKickCrashFix/info.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "HeadlessToolPermissionKickCrashFix",
+	"id": "owo.Nytra.HeadlessToolPermissionKickCrashFix",
+	"description": "A Headless Server mod that prevents tool permission kicks from causing a world crash.",
+	"category": "Bug Workaround",
+	"sourceLocation": "https://github.com/Nytra/ResoniteHeadlessToolPermissionKickCrashFix",
+	"platforms": [
+		"headless"
+	],
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/Nytra/ResoniteHeadlessToolPermissionKickCrashFix/releases/tag/v1.0.0",
+			"artifacts": [{
+				"url": "https://github.com/Nytra/ResoniteHeadlessToolPermissionKickCrashFix/releases/download/v1.0.0/HeadlessToolPermissionKickCrashFix.dll",
```

Comment:
Non-blocking: harmony id doesn't match provided id and appears like it could change easily via adding/removing authors since you build the string